packages/android_alarm_manager_plus/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.0.8
+
+- Now we can send extra data as Map to alarm manager and receive it in our callback
+
```

Comment:
Please remove (see our updated CONTRIBUTOR guidelines), we handle the changelog and version for now on.